app/src/main/java/com/onboarding/parkingsystemjava/fragment/SetParkingLotsDialog.java

```diff
@@ -0,0 +1,70 @@
+package com.onboarding.parkingsystemjava.fragment;
+
+import android.app.DialogFragment;
+import android.content.Context;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+
+import androidx.annotation.Nullable;
+
+import com.onboarding.parkingsystemjava.R;
+import com.onboarding.parkingsystemjava.mvp.contract.SetParkingContract;
+import com.onboarding.parkingsystemjava.mvp.presenter.SetParkingPresenter;
+import com.onboarding.parkingsystemjava.mvp.view.SetParkingView;
+
+public class SetParkingLotsDialog extends DialogFragment {
+
+    private SetParkingContract.SetParkingPresenter presenter;
+    private EditText input;
+    private Button okButton;
+    private Button cancelButton;
+    private static final String TAG = "SetParkingLotsDialog";
+    private OnInputListener inputListener;
+
+    public interface OnInputListener {
+        void sendInput(int input);
+    }
+
+    public static SetParkingLotsDialog newInstance() {
+        return new SetParkingLotsDialog();
+    }
+
+    @Override
+    public void onStart() {
+        super.onStart();
+        this.presenter = new SetParkingPresenter(this.inputListener, new SetParkingView(this));
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.set_parking_dialog_fragment_main_activity, container, false);
+        input = view.findViewById(R.id.edit_text_dialog_fragment_parking_input);
```

Comment:
do not use findViewById pls, use viewBinding

Reply:
done